docs/references/faq.md

```diff
@@ -9,63 +9,66 @@ Queries regarding general usage, troubleshooting, features, performance, and mor
 - Traceable answers with reduced hallucinations: You can trust RAGFlow's responses as you can view the citations and references supporting them.
 
-### 2. Which languages does RAGFlow support?
+### Which embedding models can be deployed locally?
+
+RAGFlow offers two Docker image editions, `dev-slim` and `dev`:  
+  
+- `infiniflow/ragflow:dev-slim` (default): The RAGFlow Docker image without embedding models.  
+- `infiniflow/ragflow:dev`: The RAGFlow Docker image with embedding models including:
+  - Built-in embedding models:
+    - `BAAI/bge-large-zh-v1.5` 
+    - `BAAI/bge-reranker-v2-m3`
+    - `maidalun1020/bce-embedding-base_v1`
+    - `maidalun1020/bce-reranker-base_v1`
+  - Embedding models that will be downloaded once you select them in the RAGFlow UI:
+    - `BAAI/bge-base-en-v1.5`
+    - `BAAI/bge-large-en-v1.5`
+    - `BAAI/bge-small-en-v1.5`
+    - `BAAI/bge-small-zh-v1.5`
+    - `jinaai/jina-embeddings-v2-base-en`
+    - `jinaai/jina-embeddings-v2-small-en`
+    - `nomic-ai/nomic-embed-text-v1.5`
+    - `sentence-transformers/all-MiniLM-L6-v2`
+
+### Do you offer an API for integration with third-party applications?
 
-English, simplified Chinese, traditional Chinese for now.
-
-### 3. Which embedding models can be deployed locally?
-
-- BAAI/bge-large-zh-v1.5
-- BAAI/bge-base-en-v1.5
-- BAAI/bge-large-en-v1.5
-- BAAI/bge-small-en-v1.5
-- BAAI/bge-small-zh-v1.5
-- jinaai/jina-embeddings-v2-base-en
-- jinaai/jina-embeddings-v2-small-en
-- nomic-ai/nomic-embed-text-v1.5
-- sentence-transformers/all-MiniLM-L6-v2
-- maidalun1020/bce-embedding-base_v1
+The corresponding APIs are now available. See the [RAGFlow HTTP API Reference](./http_api_reference.md) or the [RAGFlow Python API Reference](./python_api_reference.md) for more information.
 
 ## Performance
 
-### 1. Why does it take longer for RAGFlow to parse a document than LangChain?
+### Why does it take longer for RAGFlow to parse a document than LangChain?
 
-We put painstaking effort into document pre-processing tasks like layout analysis, table structure recognition, and OCR (Optical Character Recognition) using our vision model. This contributes to the additional time required. 
+We put painstaking effort into document pre-processing tasks like layout analysis, table structure recognition, and OCR (Optical Character Recognition) using our vision models. This contributes to the additional time required.
 
-### 2. Why does RAGFlow require more resources than other projects?
+### Why does RAGFlow require more resources than other projects?
 
 RAGFlow has a number of built-in models for document structure parsing, which account for the additional computational resources.
 
 ## Feature
 
-### 1. Which architectures or devices does RAGFlow support?
+### Which architectures or devices does RAGFlow support?
 
 Currently, we only support x86 CPU and Nvidia GPU.
```

Comment:
officially support x86 CPU and nvidia GPU. We also test it on ARM64 platform. But we don't maintain the ARM version image.